R/data.r

```diff
@@ -71,16 +71,17 @@
 
 #' stopword lists including parts-of-speech
 #'
-#' Stopword lists that include specific parts of speech, created by
-#' Kohei Watanabe.
+#' Stopword lists that include specific parts of speech, maintained by Kohei
+#' Watanabe.
 #'
-#' These are multi-level lists, in the original data.  If you wish to
-#' use them as lists, please access the data object directly.
+#' These are multi-level lists, in the original data.  If you wish to use them
+#' as lists, please access the data object directly.
 #' @usage NULL
-#' @section Usage:
-#' `stopwords(language = "en", source = "marimo")`
-#' @source
-#'   KOHEI TO ADD
+#' @section Usage: `stopwords(language = "en", source = "marimo")`
```

Comment:
@koheiw Would it be worth adding more detail about the structure of the nested list components? Also what about naming each level? (as per https://github.com/quanteda/stopwords/issues/22#issuecomment-604380426)

Reply:
OK, I will add names to `number` and `time` first.

Reply:
Great. I had to modify the input yml files slightly to make them readable - I don't know what exactly this was, but it was in the .ja and .he files using `yaml::read_yaml()` - so I suggest you either test it with those on your side or modify the source files directly in https://github.com/quanteda/stopwords/tree/add-marimo/data-raw/marimo with the names.